diff --git a/config.h b/config.h
index 3e07365..1f3ff6e 100644
--- a/config.h
+++ b/config.h
@@ -1,16 +1,11 @@
-#define CMDLENGTH 45
-#define DELIMITER "  "
-#define CLICKABLE_BLOCKS
+#define CMDLENGTH 10000
+#define DELIMITER "   "
 
 const Block blocks[] = {
-	BLOCK("sb-mail",    1800, 17),
-	BLOCK("sb-music",   0,    18),
-	BLOCK("sb-disk",    1800, 19),
-	BLOCK("sb-memory",  10,   20),
-	BLOCK("sb-loadavg", 5,    21),
-	BLOCK("sb-mic",     0,    26),
-	BLOCK("sb-record",  0,    27),
-	BLOCK("sb-volume",  0,    22),
-	BLOCK("sb-battery", 5,    23),
-	BLOCK("sb-date",    1,    24)
+	BLOCK("dwm_bar net",  3,  6),
+	BLOCK("dwm_bar tmp",  3,  5),
+	BLOCK("dwm_bar cpu",  3,  4),
+	BLOCK("dwm_bar ram",  3,  3),
+	BLOCK("dwm_bar vol",  0,  2),
+	BLOCK("dwm_bar clk",  60, 1)
 };
