diff --git a/config.def.h b/config.def.h
index 710f7f0..8ce9e8d 100644
--- a/config.def.h
+++ b/config.def.h
@@ -58,7 +58,7 @@ enum {
 #ifdef _THUMBS_CONFIG
 
 /* thumbnail sizes in pixels (width == height): */
-static const int thumb_sizes[] = { 32, 64, 96, 128, 160 };
+static const int thumb_sizes[] = { 32, 64, 96, 128, 160, 172, 204, 236, 268, 300 };
 
 #if THUMBS_PADDING_PATCH
 const int THUMB_BORDERS[] = { 2, 2, 2, 2, 2 };
@@ -67,7 +67,7 @@ const int THUMB_PADDING = 2;
 #endif // THUMBS_PADDING_PATCH
 
 /* thumbnail size at startup, index into thumb_sizes[]: */
-static const int THUMB_SIZE = 3;
+static const int THUMB_SIZE = 7;
 
 #endif
 #ifdef _MAPPINGS_CONFIG
